src/design/components/organisms/Table/Table.tsx

```diff
@@ -204,7 +204,6 @@ const TableContainer = styled.div`
       .table-row > div:nth-child(3) {
         z-index: 10;
         position: sticky;
-        left: 340px;
```

Comment:
Imposes strict width on initial/title column (documents)

Can someone check why those are here and should we remove them all now that we have column width changeable?

@Davy-c @Rokt33r